### ISSUE TITLE
cors module does not seem to function

### ISSUE THREAD

**Issue by n1ru4l** [talentlessguy/tinyhttp#256]
**Describe the bug**

I tried migrating from an express setup to tinyhttp. Tried replacing `cors` with `@tinyhttp/cors` as a drop-in replacement. The CORS check fails and no request can be made.

**To Reproduce**

1. clone https://github.com/n1ru4l/graphql-bleeding-edge-playground
2. run `yarn add -E @tinyhttp/cors`
3. In `server/main.ts`: replace `cors` import with `@tinyhttp/cors` import
4. start server `yarn server:start` and frontend `yarn start`
5. visit `http://localhost:3000/graphql`
6. open network tab
7. select `GraphQL over HTTP` in Dropdown on Graphiql
8. inspect network tab 

**Expected behavior**

The response should be sent from the api to the frontend, instead the cors check fails.

**Versions**

* `node`: 16
* `@tinyhttp/app`: 1.3.7
* `@tinyhttp/cors`: 1.3.0

**Comment by talentlessguy** [comment]
![image](https://user-images.githubusercontent.com/35937217/120664373-8d9c4f00-c493-11eb-8ec5-fca10c3cfa8d.png)

it seems to be working?

**Comment by talentlessguy** [comment]
looks like the bug is this:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://localhost:4000/graphql. (Reason: header ‘content-type’ is not allowed according to header ‘Access-Control-Allow-Headers’ from CORS preflight response)
```

**Comment by talentlessguy** [comment]
@n1ru4l would you like to submit default parameter for `@tinyhttp/cors` for `Content-Type` in allowed headers?

https://github.com/talentlessguy/tinyhttp/blob/afd0a11d38e9b6741b6d1f3d25e5f12538e44876/packages/cors/src/index.ts#L22

otherwise, the fix is this:

```js
app.use(cors({ allowedHeaders: ['content-type'] }))
```

**Comment by talentlessguy** [comment]
i think it's now available in `@tinyhttp/cors@1.3.2`, pls check @n1ru4l
